domain/models/event.js

```diff
@@ -1,15 +1,66 @@
-'use strict'
+ 'use strict'
 
 module.exports = (sequelize, DataTypes) => {
   const Event = sequelize.define('Event', {
-    title: DataTypes.STRING,
-    startDate: DataTypes.DATEONLY,
-    endDate: DataTypes.DATEONLY,
-    startTime: DataTypes.TIME,
-    endTime: DataTypes.TIME,
-    type: DataTypes.STRING,
-    information: DataTypes.STRING(32767),
-    kuksaEventId: DataTypes.INTEGER // ID of the corresponding Kuksa event. Null if not synced (local event).
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        len: [1,255],
+        not: ['^( |\t|\n)*$','i'], // Not whitespace-only
+      },
+    },
+    startDate: {
+      type: DataTypes.DATEONLY,
+      allowNull: false,
+      validate: {
+        isDate: true,
+      },
+    },
+    endDate: {
+      type: DataTypes.DATEONLY,
+      allowNull: false,
+      validate: {
+        isDate: true,
+      },
+    },
+    startTime: {
+      type: DataTypes.TIME,
+      allowNull: false,
+      validate: {
+      },
+    },
+    endTime: {
+      type: DataTypes.TIME,
+      allowNull: false,
+      validate: {
+      },
+    },
+    type: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        // TODO: should there be isIn validation for type? 
+        // if there was isIn validation, it could cause problems 
+        // if some kuksa event had a type that is not listed here...
+        // isIn: [['leiri', 'retki']],
+        not: ['^( |\t|\n)*$','i'], // Not whitespace-only
+        len: [1,255],
+      },
+    },
+    information: {
+      type: DataTypes.STRING(32767),
+      allowNull: false,
+      validate: {
+        len: [0,32767],
```

Comment:
Does the length validation do something extra aside from the database type already having a limit?